src/cheri/xycheriot.adoc

```diff
@@ -0,0 +1,511 @@
+:cheriot_ext_name: Xycheriot
+
+:cheriot_auicgp_name: AUICGP
+:cheriot_auicgp_name_lc: auicgp
+
+:cheriot_gettop_name: YTOPR
+:cheriot_setbounds_round_down_name: YBNDSRDW
+:cheriot_setbounds_big_imm_name: YBNDSWBI
+
+:cheriot_ot_x_min: 1
+:cheriot_sentry_ot_min: 1
+:cheriot_sentry_ot_bd: 4
+:cheriot_sentry_ot_be: 5
+:cheriot_sentry_ot_fd: 2
+:cheriot_sentry_ot_fe: 3
+:cheriot_sentry_ot_fi: 1
+:cheriot_sentry_ot_max: 5
+:cheriot_ot_x_max: 7
+:cheriot_ot_absent: 8
+:cheriot_ot_nx_min: 9
+:cheriot_ot_nx_max: 15
+:cheriot_ot_max: 15
+
+[appendix]
+[#section_cheriot,reftext="{cheriot_ext_name}"]
+== "{cheriot_ext_name}" Extension and Capabilities
+
```

Comment:
```suggestion
IMPORTANT: This chapter is not part of the to-be ratified package. We include it in this specification since it is important to illustrate how the base RVY ISA can be augmented and why certain extension hooks exist in the current prose.

```

Maybe something like this?